lib/src/rules/unsafe_unicode.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+
+import '../analyzer.dart';
+import '../util/unicode_utils.dart';
+
+const _desc = r'Do not use bidirectional Unicode text.';
+
+const _details = r'''
+
+**AVOID** using bidirectional Unicode text.
```

Comment:
I like that you link to the CVE publication but is there some other link to info about bidirectional text we can point to here? Or an example?

Reply:
I added a spec link but an example would be welcome.